src/main/java/com/launchacademy/giantleap/models/MarvelCharacter.java

```diff
@@ -22,16 +22,16 @@
 @Setter
 @Getter
 @NoArgsConstructor
-public class Character {
+public class MarvelCharacter {
 
   @Id
-  @SequenceGenerator(name="character_generator", sequenceName="characters_id_seq", allocationSize = 1)
-  @GeneratedValue(strategy= GenerationType.SEQUENCE, generator="character_generator")
-  @Column(name="id", nullable=false, unique=true)
+  @SequenceGenerator(name = "character_generator", sequenceName = "characters_id_seq", allocationSize = 1)
+  @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "character_generator")
+  @Column(name = "id", nullable = false, unique = true)
```

Comment:
you should update your sequence generator and generated values to reflect the updated class name